<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardGameClass</class>
 <widget class="QMainWindow" name="CardGameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>CardGame</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startGameButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="waitForPlayerLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Waiting for another player</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="placeholderText">
     <string>Set Your Name</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChat">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Chat</string>
    </property>
    <addaction name="actionGlobalChat"/>
    <addaction name="actionLocalChat"/>
   </widget>
   <addaction name="menuChat"/>
  </widget>
  <action name="actionGlobalChat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GlobalChat</string>
   </property>
  </action>
  <action name="actionLocalChat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LocalChat</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cardgame.qrc"/>
 </resources>
 <connections/>
</ui>
